util/globals.py

```diff
@@ -48,6 +48,30 @@ def get_macros(self, ioc_name):
                 macros[key] = value
         return macros
 
+    def get_values_of_macro(self, macro_name):
+        """
+        Get the values of all macros of a given name.
+        :param macro_name: The name of the macro to search for
+        :return:  A dictionary of all instances of the matching values and the values.
+        """
+        lines = self.get_lines()
+        macros = dict()
+        for line in lines:
+            if line.__contains__(macro_name):
+                line = line.split("__")[1]
+                key, value = line.split("=")
+                macros[key] = value
+        return macros
```

Comment:
I'm not sure this works if a line is commented out. May need to pre-filter the file by removing anything after a `#` character and up to the end of a line. Also check what happens if you have something like `SIMULATE=1` on it's own in a globals file (this is valid syntax and will come up on one or two instruments)